tests/integration_tests/config_tests.py

```diff
@@ -137,6 +137,9 @@ def test_python_date_format_by_column_name(self):
         self.assertEqual(dms_col.python_date_format, "invalid")
 
     def test_expression_by_column_name(self):
```

Comment:
This tests only fails in SQLite, for some reason. I decided to skip it.

Reply:
I was not able to reproduce the failing SQLite tests no matter what — I even tried https://github.com/nektos/act and running the containers manually. So I rewrote them as proper unit tests. :-P